pipeline/terraform0.12/stack/dynamo.tf

```diff
@@ -0,0 +1,104 @@
+# Graph table
+
+resource "aws_dynamodb_table" "matcher_graph_table" {
+  name     = "${local.namespace_hyphen}_works-graph"
+  hash_key = "id"
+
+  attribute {
+    name = "id"
+    type = "S"
+  }
+
+  attribute {
+    name = "componentId"
+    type = "S"
+  }
+
+  billing_mode = "PAY_PER_REQUEST"
+
+  global_secondary_index {
+    name            = "work-sets-index"
+    hash_key        = "componentId"
+    projection_type = "ALL"
+  }
+}
+
+data "aws_iam_policy_document" "graph_table_readwrite" {
+  statement {
+    actions = [
+      "dynamodb:UpdateItem",
+      "dynamodb:PutItem",
+      "dynamodb:BatchGetItem",
+      "dynamodb:GetItem",
```

Comment:
minor: I like to sort lists like this, so as they get longer it's easier to see what they contain.

Reply:
This?